projects/gcsweb/pipeline.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: v1
+kind: Template
+objects:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    name: gcsweb
+- kind: BuildConfig
+  apiVersion: v1
+  metadata:
+    name: gcsweb-binary
+  spec:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: gcsweb:binaries
+    source:
+      git:
+        uri: https://github.com/openshift/test-infra.git
+      contextDir: gcsweb
+      type: Git
+      dockerfile: |
+        FROM openshift/origin-release:golang-1.7
+
+        ENV ARCH=amd64 VERSION=v1.0.4 PKG=k8s.io/test-infra/gcsweb
```

Comment:
I don't feel strong. If we can change something w/o forcing them to change their flows, then it's OK.